spec/facades/weather_facades_spec.rb

```diff
@@ -12,7 +12,7 @@
       expect(data.humidity).to be_a(Integer)
       expect(data.description).to be_a(String)
       expect(data.visibility).to be_a(Integer)
-      expect(data.wind_speed).to be_a(Float)
+      expect(data.wind_speed).to be_a(Float).or be_a(Integer)
```

Comment:
oh nice. I saw this earlier, I didn't know we could give two possibilities like that.